src/App.js

```diff
@@ -0,0 +1,83 @@
+import React, { useEffect } from 'react';
+import { Routes, Route, Navigate } from 'react-router-dom';
+import './App.css';
+import { useDispatch, useSelector } from 'react-redux';
+import { AnimatePresence, motion } from 'framer-motion';
+import Navbarlist from './components/navbarlist/Navbarlist';
+import Home from './components/home/Home';
+import Booking from './components/booking/Booking';
+import Reservation from './components/reservation/Reservations';
+import Detailed from './components/home/detailed/Detailed';
+import DeleteHome from './components/delete_home/DeleteHome';
+import AddHome from './components/add_home/AddHome';
+import Login from './components/login/Login';
+import {
+  loginset,
+  loginreset,
+  usernamereset,
+} from './components/login/loginSlicer';
+import Signup from './components/signup/Signup';
+import './components/home/home.css';
+
+function App() {
```

Comment:
- [ ] When I run the app and try to create a new user from scratch, this happens:

![image](https://github.com/asharanjith/stay-a-while-front-end/assets/57421823/ad1c19bc-f21e-40bc-8cab-7f8103943f27)

![image](https://github.com/asharanjith/stay-a-while-front-end/assets/57421823/80ef7a8b-785f-4e0a-9d38-c97d4c971469)

The server is rejecting my registration because the secret key and/or token does not exist (this made some tests fail in the backend too).

It may work on your end because your browser has the secret key/token stored already, but not for new users (you can try it on a private window, or by eliminating the local storage in the application tab in your browser).

Kindly fix this bug, so your users can run and test your app.

_note: This means I wasn't able to test your app's functionality.  This will be tested on the next review._

Reply:
@Meltrust But the new user who want to run it locally. need to create a master key for their own we can't share our master key to the user. It is a private key

Reply:
Please delete config/credentials.yml.enc and run the bellow lin in command prompt
EDITOR=code rails credentials:edit (for vs code)

Reply:
> Please delete config/credentials.yml.enc and run the bellow lin in command prompt EDITOR=code rails credentials:edit (for vs code)

Hey @rbhatt1999 👋 I performed those steps during the review.  However, It didn't work for me.


I can do it again now:

![image](https://github.com/asharanjith/stay-a-while-front-end/assets/57421823/daac2ee3-cc2d-426a-b309-1f418ee4f4c3)

Reply:
Please first delete the credentials files then only it can work

Reply:
Also please use vs cdoe since this command can work only in vs code

Reply:
I think I found the issue look:

![image](https://github.com/asharanjith/stay-a-while-front-end/assets/57421823/192f1878-0c53-4c53-b563-793b2741b643)

And then:

![image](https://github.com/asharanjith/stay-a-while-front-end/assets/57421823/64dbef7c-601a-4c89-8741-a5aafaf2c45a)

Reply:
Yes it will not be available for you. It is called secret key since it should not be shared @Meltrust

Reply:
2 more steps need to do after creating master key

Reply:
use bellow comment to add the secret key in credential file
1 Run below command 
`EDITOR="code --wait" bin/rails credentials:edit`
2. New file will be open then,
3. Add a key value pair there for `jwt` as shown in below screen shot (please note I'm hiding  the value for safety reason)
![image](https://github.com/asharanjith/stay-a-while-front-end/assets/112093825/b6903bf2-87ad-4fb9-8036-1fb003404cee)

Reply:
@Meltrust let me know if you are still not able to create secret key for you we can connect though a meeting

Reply:
@rbhatt1999 Thanks for steps for adding the jwt secret key!

Now, as we discussed on Slack, I highly suggest you add these instructions to your readme so all your users are able to run your app smoothly and easily. 👍